extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxRenderServiceHelper.php

```diff
@@ -57,9 +57,11 @@ public function extendImageData( $data, $width ) {
 		// title param is provided through reference in WikiaFileHelper::getFileFromTitle
 		$title = $data[ 'name' ];
 		$file = \WikiaFileHelper::getFileFromTitle( $title );
-		if ( !$file || !$file->exists() ) {
+
+		if ( !$file || !$file->exists() || !in_array($file->media_type, array(MEDIATYPE_BITMAP, MEDIATYPE_DRAWING))) {
```

Comment:
What do you think about using `$file->getMediaType()`? 
https://github.com/Wikia/app/blob/dev/includes/filerepo/file/LocalFile.php#L584